src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -22,19 +22,25 @@ partial class mincore
         [DllImport(Libraries.CoreFile_L1, EntryPoint = "GetFullPathNameW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
         private static extern int GetFullPathNamePrivate(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero);
 
-        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero)
+        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer)
```

Comment:
Thanks. I'll give it a read.

I'm assuming that you're saying that we should have only pure P/Invokes and no other methods in interop. Do you have proposals for discoverable places to put methods that wrap proper invocation per the documentation?